product-cli/cmd/cluster/aws/create.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"context"
+	"github.com/spf13/cobra"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1beta1"
+	hypershiftaws "github.com/openshift/hypershift/cmd/cluster/aws"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "aws",
+		Short:        "Creates basic functional HostedCluster resources on AWS",
+		SilenceUsage: true,
+	}
+
+	opts.AWSPlatform = core.AWSPlatformOptions{
+		AWSCredentialsFile: "",
+		Region:             "us-east-1",
+		InstanceType:       "",
+		RootVolumeType:     "gp3",
+		RootVolumeSize:     120,
+		RootVolumeIOPS:     0,
+		EndpointAccess:     string(hyperv1.Public),
+	}
+
+	cmd.Flags().StringVar(&opts.AWSPlatform.AWSCredentialsFile, "aws-creds", opts.AWSPlatform.AWSCredentialsFile, "Filepath to an AWS credentials file.")
+	cmd.Flags().StringVar(&opts.AWSPlatform.IAMJSON, "iam-json", opts.AWSPlatform.IAMJSON, "Filepath to a file containing IAM information for the cluster; if this is not specified, IAM will be automatically created.")
+	cmd.Flags().StringVar(&opts.AWSPlatform.Region, "region", opts.AWSPlatform.Region, "The region to use for created AWS infrastructure.")
+	cmd.Flags().StringSliceVar(&opts.AWSPlatform.Zones, "zones", opts.AWSPlatform.Zones, "The availability zones in which NodePools will be created.")
+	cmd.Flags().StringVar(&opts.AWSPlatform.InstanceType, "instance-type", opts.AWSPlatform.InstanceType, "The instance type for the NodePool machines.")
+	cmd.Flags().StringVar(&opts.AWSPlatform.RootVolumeType, "root-volume-type", opts.AWSPlatform.RootVolumeType, "The type of the root volume for the NodePool machines (ex. gp3, io2).")
```

Comment:
Is there a way to stay consistent between what we expose in nodepool command and cluster command for NodePool choices?
E.g both sharing the same AWSPlatform.NodePool struct or so?